rust-toolchain.toml

```diff
@@ -1,2 +1,2 @@
 [toolchain]
-channel = "1.86"
+channel = "1.87"
```

Comment:
I remember that there is a second instance of the rust version to bump, a dockerfile or something

Reply:
found it